.github/workflows/main.yaml

```diff
@@ -0,0 +1,31 @@
+name: TEST
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+  workflow_dispatch:
+
+jobs:
+  test:
+
+    strategy:
+      matrix:
+        platform: [ubuntu-latest, macos-latest, windows-latest]
```

Comment:
Maybe we don't need macos or windows for now? Or we can have them too, if the tests don't take very long and they pass. Let's see how much they prolong the CI times...

Reply:
`ubuntu-latest` and `macos-latest` seem to run quite fast but I don't know if windows are even running? So should we remove it?